Packs/CVESearch/Integrations/CVESearch/CVESearch.py

```diff
@@ -0,0 +1,112 @@
+import urllib3
+from typing import Dict, Any, Tuple, List, Union
+
+from CommonServerPython import *
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+class Client(BaseClient):
+    """
+    Client will implement the service API, and should not contain any Demisto logic.
+    Should only do requests and return data.
+    """
+
+    def __init__(self, base_url: str):
+        header = {
+            'Content-Type': 'application/json',
+            'Accept': 'application/json'
+        }
+        super().__init__(base_url=base_url, headers=header)
+
+    def http_request(self, suffix: str) -> Union[Dict[str, Any], List[Dict[str, Any]]]:
+        """Connects to api and Returns response.
+           Args:
+               suffix :The API endpoint.
+           Returns:
+               response from the api.
+           """
+        return self._http_request(method="GET", url_suffix=suffix)
+
+
+def cve_to_context(cve):
+    return {
+        "ID": cve.get("id", ""),
+        "CVSS": cve.get("cvss", "0"),
+        "Published": cve.get("Published", "").rstrip("Z"),
+        "Modified": cve.get("Modified", "").rstrip("Z"),
+        "Description": cve.get("summary", '')
+    }
+
+
+def test_module(client: Client, *_):
+    """
+    Returning 'ok' indicates that the integration works like it is supposed to. Connection to the service is successful.
+    Returns:
+        'ok' if test passed, anything else will fail the test.
+    """
+    res = client.http_request(suffix="/last")
+    if res:
+        return "ok", None, None
+    raise Exception('Error occurred while trying to query the api.')
+
+
+def cve_latest_commnad(client: Client, *_) -> Tuple[
+        Any, Dict[str, List[Any]], Union[Dict[str, Any], List[Dict[str, Any]]]]:
+    ec = []
+    human_readable = []
```

Comment:
no need to initialize `ec` and `human_readable` here, just build `data` like you do in the loop and initalize them after